docs/hugo/content/guide/aso-controller-settings-options.md

```diff
@@ -0,0 +1,143 @@
+as well as other operator pod options. 
+
+The supported options are:
+
+
+### AZURE_SUBSCRIPTION_ID
+
+Azure subscription the operator will use for ARM communication if 
+no [more specific](multitenant-deployment.md#single-operator-multitenancy-default-recommended)
+credential is specified at the per-resource or per-namespace scope.
+
+**Format:** `GUID`
+
+**Example:** `00000000-0000-0000-0000-000000000000`
+
+**Required**: True
+
+This may be set to empty string to configure no global credential.
+
+### AZURE_TENANT_ID
+
+Azure tenantID the operator will use for ARM communication if
+no [more specific](multitenant-deployment.md#single-operator-multitenancy-default-recommended)
+credential is specified at the per-resource or per-namespace scope.
+
+**Format:** `GUID`
+
+**Example:** `00000000-0000-0000-0000-000000000000`
+
+**Required**: True
+
+This may be set to empty string to configure no global credential.
+
+### AZURE_CLIENT_ID
+
+Azure clientID the operator will use for ARM communication if
+no [more specific](multitenant-deployment.md#single-operator-multitenancy-default-recommended)
+credential is specified at the per-resource or per-namespace scope.
+
+**Format:** `GUID`
+
+**Example:** `00000000-0000-0000-0000-000000000000`
+
+**Required**: True
+
+This may be set to empty string to configure no global credential.
+
+### AZURE_CLIENT_SECRET
+
+The secret associated with the client to use if
+no [more specific](multitenant-deployment.md#single-operator-multitenancy-default-recommended)
+credential is specified at the per-resource or per-namespace scope.
+
+**Format:** `String`
+
+**Required**: False
+
+### AZURE_SYNC_PERIOD
+
+AZURE_SYNC_PERIOD is the frequency at which resources are re-reconciled with Azure when
+there have been no triggering changes in the Kubernetes resources.
+This sync exists to detect and correct changes that happened in Azure that Kubernetes is not aware about.
+BE VERY CAREFUL setting this value low - even a modest number of resources can cause
+subscription level throttling if they are re-synced frequently. If nil or empty (`""`), sync period defaults to `1h`.
+
+**Format:** `duration string`
+
+**Example:** `"1h"`, `"15m"`, or `"60s"`. See https://pkg.go.dev/time#ParseDuration for more details.
+
+**Required**: False
+
+### AZURE_OPERATOR_MODE
+
+AZURE_OPERATOR_MODE determines whether the operator should run _watchers_, _webhooks_ or _both_ (default). An empty string, or any unrecognized value, means _both_.
+
+**Format:** `webhooks|watchers|both`
+
+**Example:** `"webhooks"`, `"watchers"` or `"both"`
+
+**Required**: False
+
+### AZURE_TARGET_NAMESPACES
+
+AZURE_TARGET_NAMESPACES lists the namespaces the operator will watch for Azure resources (if the mode includes running watchers). 
+If it's empty the operator will watch all namespaces.
+
+**Format:** `string` (comma-separated namespace names)
```

Comment:
Do we tolerate trailing spaces after the `,` ?

If we do, mention that; if we don't, let's log an issue so we do in the next release.

Reply:
We do

Reply:
We do. Updated docs